scripts/k8s_release.py

```diff
@@ -87,12 +83,8 @@ def remove_obsolete_prereleases():
         help="If set, returns the git branch name of the pre-release instead of the tag.",
         action="store_true",
     )
-    subparsers.add_parser("remove_obsolete_prereleases")
 
     kwargs = vars(parser.parse_args())
     f = locals()[kwargs.pop("subparser")]
     out = f(**kwargs)
-    if isinstance(out, (list, tuple)):
-        print(",".join(out))
-    else:
-        print(out or "")
+    print(out or "")
```

Comment:
I'm not sure i've ever looked at this line before.  There's a story here for sure